tailfs/tailfsimpl/tailfs_test.go

```diff
@@ -28,11 +28,11 @@ import (
 const (
 	domain = `test$%domain.com`
 
-	remote1 = `remote$%1`
-	remote2 = `_remote$%2`
-	share11 = `share$%11`
-	share12 = `_share$%12`
-	file111 = `file$%111.txt`
+	remote1 = `rem ote$%1`
```

Comment:
Maybe for future consideration, we should exercise more corners: Tabs, carriage returns, newlines, and probably some of the other Unicode spaces (though then we also get into the encoding issue, which is a tire fire).